scripts/set_assignees.py

```diff
@@ -70,16 +70,10 @@ def process_pr(gh, maintainer_file, number):
     all_areas = set()
     fn = list(pr.get_files())
 
-    manifest_change = False
     for changed_file in fn:
         if changed_file.filename in ['west.yml','submanifests/optional.yaml']:
-            manifest_change = True
             break
 
-    # one liner PRs should be trivial
-    if pr.commits == 1 and (pr.additions <= 1 and pr.deletions <= 1) and not manifest_change:
-        labels = {'Trivial'}
```

Comment:
How about we change this from 'Trivial' to 'One Liner', so it is just provided as a hint but still has to go through the normal review process. Filtering on those would help identify trivial ones and low hanging fruits that can still be set manually as 'Trivial' if needed.

Reply:
Other code review systems have tags for the PR size for this sort of things, may not be a bad idea as a followup, something like XS, S, M, L, XL, then we could set trehshold here and tag, that would in my opinion be more useful than "one line or anyting else", regardless I think this should go and that be a followup.

Reply:
followup to what? the functionality is there already, we just change the label instead of removing existing logic. A followup could be expanding from 'One liner' to S,M,L,XL.

Reply:
> How about we change this from 'Trivial' to 'One Liner',

could also be renamed to `size: XS` so we can "followup" on this. This existing functionality is very useful and the size label does not mean XS is Trivial and will not be mergeable within 4 hours as we have it now.

Reply:
Done.

Reply:
should look into this: 

https://github.com/marketplace/actions/pull-request-size-labeler